.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "trailingComma": "es5",
+  "singleQuote": true,
+  "semi": false
```

Comment:
new line on end of the file not added

Reply:
fixed in https://github.com/Appnroll/puk-puk/pull/10/commits/00565ebef55b7cc8289ae54939f1c04b4f7cf80a